sections/dashboard/FuturesMarketsTable/FuturesMarketsTable.tsx

```diff
@@ -28,6 +33,7 @@ const FuturesMarketsTable: FC = () => {
 	const pastRates = useRecoilValue(pastRatesState);
 	const futuresVolumes = useAppSelector(selectMarketVolumes);
 	const accountType = useAppSelector(selectFuturesType);
+	const marketPrice = useAppSelector(selectLatestMarketPrice);
```

Comment:
This returns the price for the selected market, so duplicates the price here. I think it should be `selectedMarketPrices` and index by asset for each market.

![image](https://user-images.githubusercontent.com/10401554/207662310-fc47e551-4a9f-4a99-b1f0-f5460d2d8f74.png)